rpcd/playbooks/roles/rpc_maas/tasks/rabbitmq_user.yml

```diff
@@ -51,6 +46,6 @@
     write_priv: ".*"
     tags: "administrator"
     state: "present"
-  with_items: vhost_json
+  with_items: vhost_list.json
```

Comment:
The URI module can provide a JSON object from the contents of a response. This is making use of that capability of the URI module rather than using an intermediate task to parse the returned JSON into an object.